libraries/AP_Scripting/examples/lidar_control.lua

```diff
@@ -0,0 +1,53 @@
+-- enable use of Lidar on quadplanes only for landing, by changing RNGFN_LANDING
+
+-- bind a parameter to a variable
+function bind_param(name)
+   local p = Parameter()
+   assert(p:init(name), string.format('could not find %s parameter', name))
+   return p
+end
+
+local RNGFND_LANDING = bind_param("RNGFND_LANDING")
+
+MODE_QLAND = 20
```

Comment:
Perhaps it's a Lua question but why do we sometimes put "local" in front and sometimes don't?